lectures/cross_section.md

```diff
@@ -0,0 +1,885 @@
+
+* You can imagine this: most people have low or modest wealth but some people
+  are extremely rich.
+
+The firm size distribution is also heavy-tailed 
+({cite}`axtell2001zipf`, {cite}`gabaix2016power`).
+
+* You can imagine this too: most firms are small but some firms are enormous.
+
+The distribution of town and city sizes is heavy-tailed 
+({cite}`rozenfeld2011area`, {cite}`gabaix2016power`).
+
+* Most towns and cities are small but some are very large.
+
+
+### Why Should We Care?
+
+Heavy tails are common in economic data but does that mean they are important?
+
+The answer to this question is affirmative!
+
+When distributions are heavy-tailed, we need to think carefully about issues
+like
+
+* diversification and risk
+* forecasting
+* taxation (across a heavy-tailed income distribution), etc.
+
+We return to these points below.
+
+
+
+## Visual Comparisons
+
+Let's do some more visual comparisons to help us build intuition on the
+difference between light and heavy tails.
+
+
+The figure below shows a simulation.  (You will be asked to replicate it in
+the exercises.)
+
+The top two subfigures each show 120 independent draws from the normal
+distribution, which is light-tailed.
+
+The bottom subfigure shows 120 independent draws from [the Cauchy
+distribution](https://en.wikipedia.org/wiki/Cauchy_distribution), which is
+heavy-tailed.
+
+(light_heavy_fig1)=
+```{figure} /_static/lecture_specific/cross_section/light_heavy_fig1.png
+
+```
+
+In the top subfigure, the standard deviation of the normal distribution is 2,
+and the draws are clustered around the mean.
+
+In the middle subfigure, the standard deviation is increased to 12 and, as
+expected, the amount of dispersion rises.
+
+The bottom subfigure, with the Cauchy draws, shows a different pattern: tight
+clustering around the mean for the great majority of observations, combined
+with a few sudden large deviations from the mean.
+
+This is typical of a heavy-tailed distribution.
+
+
+## Heavy Tails in Economic Cross-Sectional Distributions
+
+TODO 
+
+- Shu please add data and plots.
+- Please use empirical CCDF, as in the macro dynamics book
+
+
+
+TODO Review exercises below --- are they all too hard for undergrads or should
+we keep some of them.
+
+```{code-cell} ipython3
+def extract_wb(varlist=['NY.GDP.MKTP.CD'], 
+               c='all', 
+               s=1900, 
+               e=2021, 
+               varnames=None):
+    if c == "all_countries":
+        #  keep countries only (no aggregated regions)
+        countries = wb.get_countries()
+        countries_code = countries[countries['region'] != 'Aggregates']['iso3c'].values
+            
+    df = wb.download(indicator=varlist, country=countries_code, start=s, end=e).stack().unstack(0).reset_index()
+    df = df.drop(['level_1'], axis=1).transpose() # set_index(['year'])
+    if varnames != None:
+        df.columns = varnames
+        df = df[1:]
+    return df
+```
```

Comment:
Thanks for your comments @jstac . Please find my solution to get rid of regions for the GDP data.

Since we only use this function once? Should we eliminate the function but use the code inside directly?

Looking forward to your comments and review.